massa-factory-worker/src/block_factory.rs

```diff
@@ -195,6 +195,16 @@ impl BlockFactoryWorker {
 
         // gather operations and compute global operations hash
         let (op_ids, op_storage) = self.channels.pool.get_block_operations(&slot);
+
+        let Ok(max_ops_usize) = usize::try_from(self.cfg.max_operations_per_block) else {
```

Comment:
I think we can just use "self.cfg.max_operations_per_block as usize" so that it panics if the config contains something really weird

Reply:
proposition statement: "having `as $INT_TYPE` throughout the code is a smell. use `.into()` or `.try_into()` instead"

Should we avoid the use of `as` instead of using the conversion methods, and setup some sort of standard to handle fallable conversion errors? (outside of scope here, of course)

Reply:
conversion use reverted